.travis.yml

```diff
@@ -29,16 +29,18 @@ matrix:
 
 before_install:
   - set -e
+  - export AMBERTOOLS=ambertools-17.0-0.tar.bz2
+  - export AMBERHOME=$HOME
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       brew tap homebrew/science;
       brew update;
       brew install netcdf fftw;
-      curl -L https://anaconda.org/AmberMD/pysander/16.0/download/osx-64/pysander-16.0-py27_1.tar.bz2 > $HOME/osx_libsander.tar.bz2;
-      tar jxf $HOME/osx_libsander.tar.bz2 lib/libsander.dylib include/sander.h;
+      curl -L http://ambermd.org/downloads/ambertools-rc/conda/osx-64/$AMBERTOOLS > $HOME/$AMBERTOOLS;
+      tar jxf $HOME/$AMBERTOOLS lib/libsander.dylib include/sander.h;
```

Comment:
You also need to extract `dat/antechamber/ATOMTYPE_CHECK.TAB` and `dat/antechamber/BONDTYPE_CHECK.TAB` to get the SYBYL/Mol2 type test to pass.